build/storage/tests/it/ptf_tests/tests/steps/ramdrive.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from tests.steps.base import TestStep
+
+
+class CreateRamdriveAndAttachAsNsToSubsystemStep(TestStep):
+    def _step(self):
+        self.result["volume_ids"] = self.platform.create_ramdrives(1)
+
+    def _assertion_after_step(self):
+        assert len(self.result["volume_ids"]) == 1
+
+
+class CreateRamdriveAndAttachAsNsToSubsystem64Step(TestStep):
+    def _step(self):
+        self.result["volume_ids"] = self.platform.create_ramdrives(64)
+
+    def _assertion_after_step(self):
+        assert len(self.result["volume_ids"]) == 64
```

Comment:
as we discussed yesterday, this number should come from config file as well since different IPUs can support different max number of virtio-blk devices

Reply:
changed